### DISCUSSION TITLE
Allow selectively enabling new features

### DISCUSSION THREAD

**Discussion by t-animal** [Keriew/augustus#601] [Ideas]
I love what you've done with the game, but for my taste it goes a bit far. It creates a new game, which is awesome and from quickly testing it, I like it, but it doesn't give me the nostalgia feeling any more.

I guess many people would only want Julius+ roadblocks and zooming. It would be nice if the game could have presets for how much changes should be applied. Something like "minor improvements" vs "complete overhaul".

**Comment by Areldir** [comment]
Converting this to a discussion thread, as the issues tab should be kept for bugs/glitches. I will let other devs comment on the potential of this idea though, as I am not qualified enough in the code to comment.

**Comment by Ouaz** [comment]
Well, Julius is a "legacy" mod, and keeps C3 as it was, but implements many QOL improvements.

Augustus is clearly stated as a mod _that intends to incorporate gameplay changes. The aim of this project is to provide enhanced, customizable gameplay to Caesar 3 using project Julius UI enhancements._

Is it possible to implement a global option to set off most of the things Augustus modified/added from vanilla? I don't think so, as it seems overly complicated, and way simpler to play Julius instead, using gatehouses. haha

**Reply by t-animal** [reply]
I understand. However, Augustus implements QOL improvements, too. Like, for example, zooming. Sure, barriers slightly change the game mechanics, but only just a bit. So does the warehouse maximum fill ratio for each product. These are things that could be in the "minor improvements" category.

Those are minimal changes that basically slightly reduce the need for micro-management in the game. I agree that for people who want to play the vanilla game, Julius is the right option, but I feel like there's probably a big group of people (me included) who may want to have a middle ground.

**Comment by Riww** [comment]
The problem as I see it, it would require tremendous amount of work to make to all the new Augustus features optional, which is better put into more development. Well basically what Ouaz already said.

**Reply by t-animal** [reply]
I understand. Is there maybe an old version that has those improvements already but not the gameplay changes that could be maintained in a separate branch or in a separate project?
